### DISCUSSION TITLE
Call from postman works fine but not from any other domain

### DISCUSSION THREAD

**Discussion by Leepian** [gotenberg/gotenberg#569] [Q&A]
I have deployed the gotenberg container on aws and want to access it from frontend using javascript. I checked it works perfect in postman and using CURL but not working using javascript/AJAX/httprequest call instead it returns cors error. If I run on same instance and call using localhost it also works fine but I need to dockerise all the modules in separate instances and domains.

Do I need to whitelist the calling domain if yes then how I can do it? I need proper solution. Thanks in advance.

**Comment by gulien** [comment]
Hello @Leepian,

You should not expose your Gotenberg instance on a public port. Best option is to have some sort of proxy between your frontend and Gotenberg instance(s).

CORS issues are not related to Gotenberg.
